tfhe/src/core_crypto/experimental/commons/traits.rs

```diff
@@ -0,0 +1,37 @@
+use crate::core_crypto::commons::traits::Container;
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum SecretKeyKind {
+    LweSecretKey,
+    GlweSecretKey,
+}
+
+pub trait SecretKey<T>: seal::Sealed + AsRef<[T]> {
```

Comment:
Is this to be able to take support `GlweSk` and `LweSk` in a single argument `impl SecretKey`?
If so, I prefer to force the user to convert the `GlweSk` into a `LweSkView` manually a it makes the APIs simpler and the explicit conversion is ok

Reply:
fair enough